Sources/PackageDescription/PackageDescription.swift

```diff
@@ -295,11 +347,10 @@ public final class Package {
     ///   - traits: The set of traits of this package.
     ///   - dependencies: The list of package dependencies.
     ///   - targets: The list of targets that are part of this package.
-    ///   - swiftLanguageVersions: The list of Swift versions with which this package is compatible.
+    ///   - swiftLanguageModes: The list of Swift language modes with which this package is compatible.
     ///   - cLanguageStandard: The C language standard to use for all C targets in this package.
     ///   - cxxLanguageStandard: The C++ language standard to use for all C++ targets in this package.
-    @_spi(ExperimentalTraits)
```

Comment:
@dempseyatgithub Cannot remove this annotation because `Traits` type is currently SPI.

Reply:
I think in general this change shouldn't need to change any existing availability annotations?...

Reply:
This broke a toolchain build, that's how I discovered that.
```
/Users/ec2-user/jenkins/workspace/swift-PR-toolchain-macos/branch-main/swiftpm/Sources/PackageDescription/PackageDescription.swift:361:21: error: cannot use struct 'Trait' here; it is SPI
359 |         providers: [SystemPackageProvider]? = nil,
360 |         products: [Product] = [],
361 |         traits: Set<Trait> = [],
    |                     `- error: cannot use struct 'Trait' here; it is SPI
362 |         dependencies: [Dependency] = [],
363 |         targets: [Target] = [],
```